src/extension/devtools/devtools.ts

```diff
@@ -45,12 +41,6 @@ const portAdapter = createPortMessageAdapter(() =>
 const clientPort = createActor<ClientMessage>(portAdapter);
 const rpcClient = createRpcClient<DevtoolsRPCMessage>(portAdapter);
 
-devtoolsMachine.subscribe(({ value }) => {
-  if (value === "connected") {
```

Comment:
This is captured in the `startRequestInterval` machine action, so we can remove the redundancy here.